docs/interfaces.md

```diff
@@ -77,11 +77,7 @@ Note: this diagram is current of v1.11.1.
     "provider": {
       "id": "string"
     },
-    "status": "string",
-    "post_to_cmr_duration": "number",
```

Comment:
Why are these fields being removed?

Reply:
The 4 `duration` fields are optional and only used by EMS metrics reporting. They do not show up on the dashboard, nor are they important for the API or other functionality. EMS reporting is rather more ancillary functionality, and since this schema is already a trimmed version of a Cumulus message first created for [this wiki page](https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=124460784), I didn't think they were relevant for a developer reading the docs. Feel free to disagree. It's not a hill I'm looking to die on.